src/rdkafka.h

```diff
@@ -3427,10 +3427,16 @@ rd_kafka_error_t *rd_kafka_sasl_set_credentials(rd_kafka_t *rk,
  * @returns a reference to the librdkafka consumer queue.
  * This is the queue served by rd_kafka_consumer_poll().
  *
- * Use rd_kafka_queue_destroy() to loose the reference.
+ * Use rd_kafka_queue_destroy() to lose the reference.
```

Comment:
Here "loose" was correct, to mean "release"

Reply:
Done, I got confused as "lose" can also mean get rid of